### ISSUE TITLE
Audio Circuit Inconsistencies

### ISSUE THREAD

**Issue by batorni** [geaz/simplyRetro-Z5#5]
In the audio circuit diagram/image, it shows the 5V + on the PAM going to the 5V Pin on the Pi, however, in the image of the completed build, it looks like that line is actually going to pin 1, 3.3V Power.

**Comment by geaz** [comment]
You are toally right. Per specification the PAM needs 5V, but it runs fine with 3.3V, too.

**Comment by batorni** [comment]
Neat!  I've got mine wired up to a 5V pin, but I'm still trying to get the darn thing to boot, so I was unsure if the audio worked or not

**Comment by geaz** [comment]
Are you using the simplyRetro image? If you have any problems with it, feel free to open an issue. Will try my best to help you.

**Comment by batorni** [comment]
Yeah, I'm using v1.3 provided here: https://github.com/geaz/simplyRetro-Z5/releases

It doesn't seem to boot via the power button, and I'm not seeing any lights from the powerboost either. 
If I plug in power to the pi directly, the powerboost turns on as well as the light on the pi turns on, but the screen isn't turning on either. I think I may have connected the screen power incorrectly. I wired the spliced USB cable to test pads 22 and 23 on the pi, rather than attaching them to test pads 1 and 6 alongside the power from the powerboost.

I need to re-check all my circuits tonight. I went all in and didn't do any intermediate testing so I'm paying for it now :)

I'll open up an issue if I'm still facing issues tonight.
